config/filter.d/postfix.conf

```diff
@@ -15,6 +15,7 @@ _daemon = postfix/(submission/)?smtp(d|s)
 failregex = ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: 554 5\.7\.1 .*$
             ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: 450 4\.7\.1 Client host rejected: cannot find your hostname, (\[\S*\]); from=<\S*> to=<\S+> proto=ESMTP helo=<\S*>$
             ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: 450 4\.7\.1 : Helo command rejected: Host not found; from=<> to=<> proto=ESMTP helo= *$
+            ^%(__prefix_line)sNOQUEUE: reject: EHLO from \S+\[<HOST>\]: 504 5\.5\.2 <\S+>: Helo command rejected: need fully-qualified hostname;
```

Comment:
> to satisfy never ending paranoia...
To satisfy it, we should review all the regexp, and rewrite the biggest part of it.
Imho, this shorter regexp (if long enough) is much more robust, so it should satisfy the paranoia rather :) In addition, it is also faster.
Do not misunderstand, I'm not against long regexp at all. Just think as programmer - the shorter, the less vulnerable.